### PR TITLE
fix: handle patterns with forward slashes

### PR DESCRIPTION
**Pull request by robert-wysocki-sparkbit**

### CODE REVIEW

**Comment by astahmer** [comment]
thanks again, same comment as the other one, just and a [changeset](https://github.com/changesets/changesets/blob/main/docs/intro-to-using-changesets.md) and you're good ! 👍

File: lib/src/utils.ts [hunk 1]
```diff
@@ -47,19 +47,19 @@
...
~                const dec: number = p1.codePointAt();
~                const hex: string = dec.toString(16);
~                // eslint-disable-next-line sonarjs/no-nested-template-literals
                 if (dec <= 0xFF) return `\\x${`00${hex}`.slice(-2)}`;
                 // eslint-disable-next-line sonarjs/no-nested-template-literals
                 return `\\u${`0000${hex}`.slice(-4)}`;
-            }
-    );
+            })
+        .replace(/\//g, "\\/");
```
[the remark this answers is not in the document]
Comment (robert-wysocki-sparkbit) [reply lib/src/utils.ts]: I'm pretty confident, it's a false positive

**Comment by astahmer** [comment]
hey, could you resolve those conflicts ? i'll merge straight after that

**Comment by robert-wysocki-sparkbit** [comment]
> hey, could you resolve those conflicts ? i'll merge straight after that

Sure thing. BTW I'm wondering whether those test be better suited for `openApiToZod.test.ts`?

**Comment by astahmer** [comment]
> Sure thing. BTW I'm wondering whether those test be better suited for openApiToZod.test.ts?

I don't have any specific opinion here, if you think this makes more sense in `openApiToZod.test.ts` you can move them it's fine 👍

**Comment by robert-wysocki-sparkbit** [comment]
I have no idea why this PR decided to close itself

**Pull request closed without merging**